packages/oc-template-handlebars-compiler/lib/compile.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const _ = require('lodash');
+const async = require('async');
+const compileServer = require('./compileServer');
+const compileStatics = require('./compileStatics');
+const compileView = require('./compileView');
+const fs = require('fs-extra');
+const getUnixUtcTimestamp = require('oc-get-unix-utc-timestamp');
+const path = require('path');
+
+// OPTIONS
+// =======
```

Comment:
Same as previous PR, can we make this list of options alphabetic and comprehensive? (ocPackage missing)